_posts/2020-01-13-serverless-express-api.md

```diff
@@ -0,0 +1,562 @@
+---
+layout: post
+title: Build & Deploy a Serverless Express API to AWS
+date: 2020-01-13
+tags: javascript tutorial serverless sam
+author: Matthew Tyler
+---
+
+# Introduction
+
+Released in 2015, AWS API Gateway allows developers to build and publish APIs that can be consumed by clients over public internet and virtual private cloud networks. At Re:Invent 2019, announced a significant addition to the service, that they have called "HTTP APIs". The HTTP API is a direct response to customers who wanted to build simple HTTP backed API's, who did not need the complete feature set of API Gateway. The new HTTP API service is perfect for hosting lamba backend applications that are written in popular HTTP Frameworks like Express, Flask, .NET etc.
```

Comment:
"At Re:Invent 2019, announced a significant" should it be "At Re:Invent 2019, AWS announced a significant"?